lib/flagship.rb

```diff
@@ -20,8 +20,12 @@ def enabled?(key)
       current_flagset.enabled?(key)
     end
 
-    def set_context(key, value)
-      default_context.__set(key, value)
+    def set_context(key_or_hash, value=nil)
+      if key_or_hash.is_a?(Hash)
+        default_context.merge!(key_or_hash)
```

Comment:
I want this to be like this.

```rb
key_or_hash.each { |k, v| default_context.__set(k, v) }
```

in `Context#__set`, key is converted to `Symbol` to avoid complicated error.
(I mean to avoid using `String` as a key.)